lib/rspec/its.rb

```diff
@@ -171,12 +171,40 @@ def should_not(matcher=nil, message=nil)
       end
     end
 
+    # Calls the #its method adding the :focus tag to metadata to allowing
+    # to be run in focus.
+    #
+    # @example
+    #
+    #   # This ...
+    #   describe Array do
+    #     fits(:size) { should eq(0) }
+    #   end
+    def fits(attribute, *options, &block)
+      opts = (Array(options) << :focus).uniq
```

Comment:
Do you need to wrap `options` with an `Array()`? Is this to dup it and prevent from an additional element being added in case the argument passed to it is reused elsewhere?

Reply:
That was my thinking, however I can remove it if you don't think it's a concern :)

Reply:
```ruby
def a(*x)
  x
end

a # => []
```

Do you think of a scenario where `options` is not an Array already?

Reply:
Can't remember why I did this, I'll make your suggested change :)